### PR TITLE
Add a SuggestionFactory and site_matrix_with_submissions fixture

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

**Comment by ta2-1** [comment]
@phlax, you called PR as "add suggestion factory...", but you add submissions to sitematrix_fixture actually. Can you rename it?

**Comment by phlax** [comment]
@ta2-1 yep the commit changed - but i didnt update title...

done

**Comment by ta2-1** [comment]
should untranslated units have initial submissions?

File: pootle_pytest/fixtures/site.py [hunk 1]
```diff
@@ -182,3 +183,77 @@ def site_matrix_with_announcements(site_matrix):
...
+    # mark OBSOLETE
+    if original_state == OBSOLETE:
+        unit.makeobsolete()
+
+
+@pytest.fixture
+def site_matrix_with_submissions(site_matrix):
+    from pootle_store.models import Unit
+
+    year_ago = datetime.now() - timedelta(days=365)
```
Comment (julen) [inline pootle_pytest/fixtures/site.py]: Even if there might not be noticeable effects in tests, I'd rather use `django.utils.timezone.now()` everywhere to better align with the rest of the app code.

**Comment by phlax** [comment]
this is now taken care of in #4458 and #4438

**Pull request closed without merging**